libcodechecker/analyze/pre_analysis_manager.py

```diff
@@ -215,7 +215,12 @@ def signal_handler(*arg, **kwarg):
         stats_in = statistics_data.get('stat_tmp_dir')
         stats_out = statistics_data.get('stats_out_dir')
 
-        statistics_collector.postprocess_stats(stats_in, stats_out)
+        statistics_collector.postprocess_stats(stats_in, stats_out,
+                                               statistics_data.get(
+                                                   'stats_min_sample_count'),
```

Comment:
If you want to add some extra security for the case where `stats_min_sample_count` is missing from the dict for some reason `statistics_data.get('stats_min_sample_count')` could be changed to `statistics_data.get('stats_min_sample_count', 20)`. `.get` can return a default value if the key was not found.